File: tkpdflight/src/main/java/com/tokopedia/flight/orderlist/data/cloud/entity/PaymentInfoEntity.java
package com.tokopedia.flight.orderlist.data.cloud.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by alvarisi on 1/23/18.
 */

public class PaymentInfoEntity {
    @SerializedName("gateway_name")
    @Expose
    private String gatewayName;
    @SerializedName("gateway_icon")
    @Expose
    private String gatewayIcon;
    @SerializedName("expire_on")
    @Expose
    private String expireOn;
    @SerializedName("transaction_code")
    @Expose
    private String transactionCode;
    @SerializedName("total_amt")
    @Expose
    private int totalAmount;
    @SerializedName("manual_transfer")
    @Expose
    private ManualTransferEntity manualTransfer;

    public String getGatewayName() {
        return gatewayName;
    }

    public String getGatewayIcon() {
        return gatewayIcon;
    }

    public String getExpireOn() {
        return expireOn;
    }

    public String getTransactionCode() {
        return transactionCode;
    }

    public ManualTransferEntity getManualTransfer() {
        return manualTransfer;
    }

    public int getTotalAmount() {
        return totalAmount;
    }
}

File: features/digital/flight/src/main/java/com/tokopedia/flight/orderlist/data/cloud/entity/PaymentInfoEntity.kt
package com.tokopedia.flight.orderlist.data.cloud.entity

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by alvarisi on 1/23/18.
 */

class PaymentInfoEntity(
        @SerializedName("gateway_name")
        @Expose
        val gatewayName: String = "",
        @SerializedName("gateway_icon")
        @Expose
        val gatewayIcon: String = "",
        @SerializedName("expire_on")
        @Expose
        val expireOn: String = "",
        @SerializedName("transaction_code")
        @Expose
        val transactionCode: String = "",
        @SerializedName("need_to_pay_amt")
        @Expose
        val needToPayAmount: Int = 0,
        @SerializedName("manual_transfer")
        @Expose
        val manualTransfer: ManualTransferEntity)
